pyserini/search/lucene/__main__.py

```diff
@@ -88,6 +88,7 @@ def define_search_args(parser):
 
     parser.add_argument('--rm3', action='store_true', help="Use RM3")
     parser.add_argument('--rocchio', action='store_true', help="Use Rocchio")
+    parser.add_argument('--usenegative', action='store_true', help="Use nonrelevant labels in Rocchio")
```

Comment:
change to `--use-negative` please.

Reply:
Will --rocchio-use-negative be better? To be more specific for rocchio reranking.